### ISSUE TITLE
Authors with the same surname (Section 8.20)

### ISSUE THREAD

**Issue by rmolina** [plk/biblatex-apa#123]
Please consider the following MWE:

```latex
\documentclass{article}
\usepackage[american]{babel}
\usepackage[style=apa]{biblatex}
\addbibresource{\jobname.bib}

\begin{filecontents}[force]{\jobname.bib}
@book{FirstBook,
    author={First Author and Second Author},
    year={2018},
    title={First Book},
    publisher={Publisher},
}
@book{SecondBook,
    author={First Author and Second Author},
    year={2020},
    title={Second Book},
    publisher={Publisher},
}
\end{filecontents}

\begin{document}
\parencite{FirstBook,SecondBook}
\printbibliography
\end{document}
```
This results in `(F. Author & Author, 2018, 2020)`. I believe this output is incorrect. 

From the APA's manual, Section 8.20:
> If the first authors of multiple references share the same surname but have different initials, include the first authors’ initials in all in-text citations, even if the year of publication differs.

In the MWE, the first author is the same in both works (i.e., same initials), and adding the initials does not help removing any ambiguity.

Further, Section 8.20 also says:
> If the first authors of multiple references share the same surname and the same initials, cite the works in the standard author–date format.

I believe this applies here.

**Comment by plk** [comment]
Which version of the style are you using? Latest is 9.13 and I can't reproduce the issue there - I get:

<img width="413" alt="Screenshot 2020-08-05 at 9 11 pm" src="https://user-images.githubusercontent.com/1220851/89454027-6506d400-d760-11ea-8b25-7610af9f90c2.png">

**Comment by rmolina** [comment]
I'm using v9.13 from an up-to-date TeX Live 2020. Here is my log from `\listfiles`:
```
 *File List*
 article.cls    2019/12/20 v1.4l Standard LaTeX document class
  size10.clo    2019/12/20 v1.4l Standard LaTeX file (size option)
   babel.sty    2020/07/13 3.47 The Babel package
american.ldf    2017/06/06 v3.3r English support from the babel system
biblatex.sty    2019/12/01 v3.14 programmable bibliographies (PK/MW)
pdftexcmds.sty    2020-06-27 v0.33 Utility functions of pdfTeX for LuaTeX (HO)
infwarerr.sty    2019/12/03 v1.5 Providing info/warning/error messages (HO)
   iftex.sty    2020/03/06 v1.0d TeX engine tests
 ltxcmds.sty    2019/12/15 v1.24 LaTeX kernel commands for general use (HO)
etoolbox.sty    2020/07/15 v2.5i e-TeX tools for LaTeX (JAW)
  keyval.sty    2014/10/28 v1.15 key=value parser (DPC)
kvoptions.sty    2019/11/29 v3.13 Key value format for package options (HO)
kvsetkeys.sty    2019/12/15 v1.18 Key value parser (HO)
  logreq.sty    2010/08/04 v1.0 xml request logger
  logreq.def    2010/08/04 v1.0 logreq spec v1.0
  ifthen.sty    2014/09/29 v1.1c Standard LaTeX ifthen package (DPC)
     url.sty    2013/09/16  ver 3.4  Verb mode for urls, etc.
  blx-dm.def    2019/12/01 v3.14 biblatex localization (PK/MW)
     apa.dbx    2020/05/30 v9.13 APA biblatex style data model
blx-compat.def    2019/12/01 v3.14 biblatex compatibility (PK/MW)
biblatex.def    2019/12/01 v3.14 biblatex compatibility (PK/MW)
standard.bbx    2019/12/01 v3.14 biblatex bibliography style (PK/MW)
     apa.bbx    2020/05/30 v9.13 APA biblatex references style
     apa.cbx    2020/05/30 v9.13 APA biblatex citation style
biblatex.cfg    
l3backend-pdfmode.def    2020-06-29 L3 backend support: PDF mode
 english.lbx    2019/12/01 v3.14 biblatex localization (PK/MW)
english-apa.lbx    2020/05/30 v9.13 APA biblatex localisation
american.lbx    2019/12/01 v3.14 biblatex localization (PK/MW)
american-apa.lbx    2020/05/30 v9.13 APA biblatex localisation
    main.bbl
 ***********
```
By the way, your output looks wrong too. From Section 8.12, I would expect `(Author & Author, 2018, 2020)` rather than `(Author & Author, 2018; Author & Author, 2020)`.

**Comment by moewew** [comment]
I get @rmolina's output with Biber 2.14, but with Biber 2.15-dev I get the desired "Author & Author"

**Comment by plk** [comment]
Ah yes, this was in fact an issue with the `ifprimaryauthor` test logic fixed in `biber` 2.15 DEV.

**Comment by rmolina** [comment]
Thanks!
